about.html

```diff
@@ -1,120 +1,112 @@
-					</a>
-					<br>
-					<br>
-					Have you ever seen those PC servers with the PlotMe plugin, and you thought "Hey, why can't I have plots on my Pocketmine server?" Well, now you can with <strong>MyPlot</strong> for PocketMine-MP and other 3rd party servers! It's easy to set up, and simple to use!
-					<br>
-					<br>
-					A list of commands and permissions can be found in the <a href="https://github.com/jasonwynn10/MyPlot/wiki">wiki</a> along with a helpful config guide.
-					<br>
-					Future commands can be found <a href="http://plotme.worldcretornica.com/cmdsandperms">here</a> on the plotme website.
-					<br>
-					<br>
-					Want to help create language resource packs? Well, you can! If you know a language that is not created yet, you can create a new language pack by opening a pull request! If the language you know is already created, you can help improve it by editing it!
-					<br>
-					The Needed Languages can be found <a href="https://github.com/jasonwynn10/MyPlot/projects/2">here</a>
-				</p>
-			</section>
-		</div>
-		<div id="footer_wrap" class="outer">
-			<footer class="inner">
-				<div id="nav" class="quicknav">
-					<div align="left" class="quicknav_wrapper">
-						<ul>
-							<li><a><b>Quick Nav</b></a>
-								<ul>
-									<li><a href="https://jasonwynn10.github.io/MyPlot">Home</a></li>
-									<li><a href="https://jasonwynn10.github.io/MyPlot/about">About MyPlot</a></li>
-									<li><a href="https://jasonwynn10.github.io/MyPlot/downloads">Downloads</a></li>
-									<li><a href="https://jasonwynn10.github.io/MyPlot/cmdsandperms">Commands and Permissions</a></li>
-									<li><a href="https://jasonwynn10.github.io/MyPlot/configurations">Configurations</a></li>
-									<li><a href="https://jasonwynn10.github.io/MyPlot/how-to-use">Setup / Upgrade</a></li>
-									<li><a href="https://jasonwynn10.github.io/MyPlot/issues">Open an issue</a></li>
-									<li><a href="https://jasonwynn10.github.io/MyPlot/contributors">Contributions</a></li>
-								</ul>
-							</li>
-						</ul>
-					</div>
-				</div>
-				<p class="copyright"align="center">MyPlot is maintained by <a href="https://github.com/jasonwynn10">jasonwynn10</a></p>
-			</footer>
-			<a href="" class="back-to-top" style="text-decoration: none; display: inline;">Top</a>
-			<script>
-				jQuery(document).ready(function () {
-					var offset = 220;
-					var duration = 250;
-					jQuery(window).scroll(function () {
-						if (jQuery(this).scrollTop() > offset) {
-							jQuery('.back-to-top').fadeIn(duration);
-						} else {
-							jQuery('.back-to-top').fadeOut(duration);
-						}
-					});
-					jQuery('.back-to-top').click(function (event) {
-						event.preventDefault();
-						jQuery('html, body').animate({scrollTop: 0}, duration);
-						return false;
-					})
-				});
-			</script>
-		</div>
-	</body>
+<head>
```

Comment:
i see, opened with npp and they were spaces. i don't feel like fixing the whole thing to tabs... so i'm just going to close this.